src/status_im2/contexts/onboarding/enter_seed_phrase/view.cljs

```diff
@@ -40,20 +39,22 @@
          (i18n/label :t/use-recovery-phrase)]
         [quo/text
          (i18n/label-pluralize (mnemonic/words-count @seed-phrase) :t/words-n)]
-        [:<>
-         [quo1/text-input
+        [rn/view
+         {:style {:height            120
+                  :margin-horizontal -20}}
+         [quo/recovery-phrase-input
           {:on-change-text      (fn [t]
                                   (reset! seed-phrase (clean-seed-phrase t))
                                   (reset! error-message ""))
+           :mark-errors?        true
+           ; TODO(@ulisesmac): error-pred must return true if a word is marked as wrong
+           :error-pred          (constantly false)
```

Comment:
It's still pending to validate if each word in the seed phrase exist, once we have a  function to validate that, we can use it in this pred